packages/coq/coq.9.0.0/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "2.0"
+synopsis: "Compatibility metapackage for Coq after the Rocq renaming"
+maintainer: ["The Rocq development team <coqdev@inria.fr>"]
+authors: ["The Rocq development team, INRIA, CNRS, and contributors"]
+license: "LGPL-2.1-only"
+homepage: "https://coq.inria.fr/"
+doc: "https://coq.github.io/doc/"
+bug-reports: "https://github.com/coq/coq/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "rocq-prover" {= version}
+  "coq-core" {= version}
+  "coq-stdlib" {= "9.0.0"}
+  "coqide-server" {= version}
+  "odoc" {with-doc}
+]
```

Comment:
Don't you want to add a post install message to the coq* packages that inform the users of the renaming?

Reply:
Good point. It's now in coq-core, which really includes compat binaries and is installed by the other "fake" compat packages.